introduction.rst

```diff
@@ -108,20 +73,32 @@ From Conan 1.0, there is a commitment to stability, not breaking user space whil
-- Bug fixes will not be considered breaking, recipes and packages relying on the incorrect behavior of such bug will be considered already broken.
+- Bug fixes will not be considered breaking, recipes and packages relying on the incorrect behavior of such bugs will be considered already broken.
 - Only documented features are considered part of the public interface of Conan. Private implementation details, and everything not included in the documentation is subject to change.
-- *conanfile.py* recipes should be defined according to the documentation in :ref:`conanfile_reference`
-- Configuration and automatic tools detection, like the detection of the default profile might be subject to change. Users are encouraged to define their configurations in profiles for repeatability. New installations of conan might use different configuration.
+- Configuration and automatic tools detection, like the detection of the default profile might be subject to change. Users are encouraged to define their configurations in profiles for repeatability. New installations of Conan might use different configurations.
 
-The compatibility is always considered forward. New APIs, tools, methods, helpers can be added in following 1.X versions. Recipes and packages created with these features will be backwards incompatible with earlier conan versions.
+The compatibility is always considered forward. New APIs, tools, methods, helpers can be added in following 1.X versions. Recipes and packages created with these features will be backwards incompatible with earlier Conan versions.
 
-This means that public repositories, like conan-center assume the use of the latest version of the Conan client, and using an older version may result in failure of packages and recipes created with a newer version of the client.
+This means that public repositories, like ConanCenter assume the use of the latest version of the Conan client, and using an older version may result in failure of packages and recipes created with a newer version of the client.
 
-Additionally, starting in version 1.6, we began the process of deprecating Python2 support.  Features already working with python2 will continue to do so, but new ones may require Python3.
-See the :ref:`deprecation notice <python2>` for more details
+Conan needs Python 3  to run. It has supported Python 2 until 1 January 2020, when it was officially deprecated by the Python maintainers. From Conan 1.22.0 release, Python 2 support is not guaranteed. See the :ref:`deprecation notice <python2>` for more details
 
 If you have any question regarding Conan updates, stability, or any clarification about this definition of stability, please report in the documentation issue tracker: https://github.com/conan-io/docs.
 
+
+
+Community
+---------
+
+Conan is being used in production by hundreds of companies like Audi, Continental, Plex, Electrolux and Mercedes-Benz and many thousands of developers around the world. 
```

Comment:
```suggestion
Conan is being used in production by hundreds of companies like [Audi](https://github.com/audi), Continental, [Plex](https://github.com/plexinc), [Electrolux](https://github.com/Electrolux) and Mercedes-Benz and many thousands of developers around the world. 
```

I'm curious, Electrolux bought Continental years ago, is it the same Continental that you referred here? I think you could by another company, since it's under Electrolux now.

Reply:
I dont think adding their repos make sense, if they don't directly show anything related to Conan there. Better leave it as-is.